packages/components/src/InputTime/InputTime.test.tsx

```diff
@@ -54,18 +51,16 @@ it("renders an initial time when given 'defaultValue'", () => {
       <label
         className="label"
         htmlFor="123e4567-e89b-12d3-a456-426655440002"
-      >
-         
-      </label>
+      />
       <input
         className="formField"
-        defaultValue="11:23"
         id="123e4567-e89b-12d3-a456-426655440002"
         onBlur={[Function]}
         onChange={[Function]}
         onFocus={[Function]}
         onKeyDown={[Function]}
         type="time"
+        value="11:23"
```

Comment:
This changed from `defaultValue` to `value` I think that's expected but just wanted to confirm since it's buried in a snapshot here.

Reply:
Yep, my understanding is that is what we want as defaultValue will essentially just transform into the value now.